playbooks/roles/forum/defaults/main.yml

```diff
@@ -20,7 +20,10 @@ FORUM_MONGO_HOSTS:
 FORUM_MONGO_TAGS: !!null
 FORUM_MONGO_PORT: "27017"
 FORUM_MONGO_DATABASE: "cs_comments_service"
-FORUM_MONGO_AUTH_DB: !!null
+FORUM_MONGO_AUTH_DB: ""
+# Must be set if user credentials are provided.
+# Can be one of :scram, :mongodb_cr, :mongodb_x509, :plain, or empty string "" if no credentials.
+FORUM_MONGO_AUTH_MECH: ":scram"
```

Comment:
⚠️ @giovannicimolin  @rusrushal13 

If this PR merges, it will change the default authentication mechanism for newly-provisioned Ironwood instances in production to SCRAM.  We can't do that until we've done the Mongo and SCRAM update in production.

Reply:
@pomegranited Yup, agree.
Let's link this PR to the PR ticket then :)